docs/examples/api-reference/client/query.py

```diff
@@ -14,14 +14,14 @@
 @mock
 def test_basic(client):
     # docsnip basic
-    from fennel.featuresets import featureset, feature, extractor
+    from fennel.featuresets import featureset, feature as F, extractor
     from fennel.lib import inputs, outputs
 
     @featureset
     class Numbers:
-        num: int = feature(id=1)
-        is_even: bool = feature(id=2)
-        is_odd: bool = feature(id=3)
+        num: int = F()
+        is_even: bool = F()
+        is_odd: bool = F()
```

Comment:
Just to confirm, we don't need to write `F()` here, right? As in, when there is no qualifier, we can simply omit this like we do with fields? cc @aditya-nambiar

Reply:
We are currently running into an issue where if the feature is not initialized with anything, we are unable to use it in an extractor ( as inputs or outputs )

Reply:
Do we have any viable paths of resolving this? Or do you expect to have to live with this at this stage?

Reply:
One option was to make inputs and outputs take string like @inputs("feature1") but this makes the code look bad.

Reply:
moving with combination then as discussed.